File: FormalBench-data/FilteredData/specifications/seed_726_per_category_10_653ade686f/java/SquarePerimeter.java
import java.io.*;
import java.lang.*;
import java.util.*;
import java.math.*;


class SquarePerimeter {
    
    /*@ public normal_behavior
      @ assignable \nothing;
      @ ensures \result == \java_math(4 * a);
      @*/
    //@ code_java_math
    public static int squarePerimeter(int a) {
        return 4 * a;
    }
}

File: FormalBench-data/FilteredData/specifications/seed_726_per_category_10_653ade686f/c/SquarePerimeter.c
#include "java_arrays.h"
#include <stdint.h>

/*@
  assigns \nothing;
  ensures \result == (int32_t)((integer)left * right);
*/
static int32_t java_mul(int32_t left, int32_t right) {
    return (int32_t)((uint32_t)left * (uint32_t)right);
}

/*@
  assigns \nothing;
  ensures \result == (int32_t)(4 * (integer)a);
*/
int32_t squarePerimeter(int32_t a) {
    return java_mul(4, a);
}
